src/pipelines/log_reg.py

```diff
@@ -33,26 +32,22 @@ def run_model(train_data, train_labels, test_data, test_labels, log_penalty='l2'
     # we create an instance of Neighbours Classifier and fit the data.
     logreg.fit(train_data, train_labels)
 
-    #Now that we have something trained we can check if it is accurate with the test set
-    pred_labels = logreg.predict(test_data)
-    perform_results = performance_metrics.get_perform_metrics(test_labels, pred_labels)
-
-    return pred_labels, perform_results
+    return logreg
 
 def main(argv):
 
-    train_data, train_target, test_data, test_target = argv[0], argv[1], argv[2], argv[3]
+    train_data, train_target = argv[0], argv[1]
 
-    if len(argv)>4:
-        args_dict = argv[4]
+    if len(argv)>2:
+        args_dict = argv[2]
     else:
         args_dict = {}
 
     print("running logistic regression...",file=sys.stderr)
 
-    predicted_labels, perform_results = run_model(train_data, train_target, test_data, test_target, **args_dict)
+    logreg = run_model(train_data, train_target, **args_dict)
 
-    return predicted_labels, perform_results
+    return logreg
 
 if __name__ == '__main__':
     if len(sys.argv) < 5:
```

Comment:
The __main__ needs be updated to reflect that we no longer expect sys.argv to have a length of 5 and the usage note doesn't need test_data or test_target anymore